addons/mail/static/src/js/client_action.js

```diff
@@ -149,10 +150,49 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
                 on_reverse_breadcrumb: this.on_reverse_breadcrumb,
             });
         },
+        "click .o_mail_chat_mobile_tab": function(event) {
```

Comment:
handlers should be defined in the bottom of the file in the 'Handlers' section, and you should only reference here the name (string), for example, '_onMobileTabClick'